modules/openapi-generator/src/main/resources/kotlin-client/libraries/multiplatform/api.mustache

```diff
@@ -66,19 +70,22 @@ import kotlinx.serialization.internal.StringDescriptor
             {{/hasFormParams}}
             {{/hasBodyParam}}
 
-        val localVariableQuery = mutableMapOf<String, List<String>>()
-        {{#queryParams}}
-        {{{paramName}}}?.apply { localVariableQuery["{{baseName}}"] = {{#isContainer}}toMultiValue(this, "{{collectionFormat}}"){{/isContainer}}{{^isContainer}}listOf("${{{paramName}}}"){{/isContainer}} }
-        {{/queryParams}}
+        {{! TODO: Don't use all the listOf() }}
+        val localVariableQuery = mutableMapOf<String, List<String>?>(
+            {{#queryParams}}
+            "{{baseName}}" to {{#isContainer}}{{{paramName}}}?.let { toMultiValue(it, "{{collectionFormat}}") }{{/isContainer}}{{^isContainer}}listOf("${{{paramName}}}"){{/isContainer}}{{#hasMore}},{{/hasMore}}
```

Comment:
This seems to have to handle the special case of a map, which has to be split up into multiple query parameters, but I don't know how to do that

Reply:
Are you looking for `isMapContainer` for map?

Reply:
Yeah, thanks

Reply:
Is there documentation on which exist somewhere?